001-IntroToKubernetes/Coach/09-helm.md

```diff
@@ -4,12 +4,71 @@
 
 ## Notes & Guidance
 - You should deliver a demo in addition to the lecture to show what a working Helm chart looks like.
+  - Consider using the `app-languages-helm-chart` helm chart provided in the Coach Solutions folder for Challenge 9 as one that can be demoed during the lecture.  
 - You should provide guidance in the challenge as to which values should be parameterized in the Helm templates.
-- **NOTE:** Helm 3 does not create namespaces with yaml files in the chart.
-	- The YAML files in the Student Resources folder provides a YAML file that creates a whatthehack namespace. 
-	- Students can not include the namespace YAML when creating the helm chart
-	- They have to create the namespace by using the **--create-namespace** flag on the helm CLI, eg:
-		- `helm install myRelease app-languages --create-namespace whatthehack`
-- Consider using the helm chart provided in the Coach Solutions folder for Challenge 9 as one that can be demoed during the lecture.  
+	- At a minimum make sure the version label is parameterized (ie: v1, v2, etc) so that students can install different versions of the app using `helm install` easily.
+- **NOTE:** Helm 3 does not automatically create namespaces.
+	- The YAML files in the Student Resources folder provides a YAML file that creates a whatthehack namespace.
+    	- This should only be used when installing the app manually during the first part of the challenge.
+- **NOTE:** When creating their helm charts make sure that students are handling namespaces properly:
+	- Students should NOT include the namespace YAML in the helm chart
+	- The application's YAML files should NOT include a namespace, we will leave namespace creation up to helm (see below).
+
+## Helm Commands 
+In this section we'll lay out all the commands needed to create the chart using the `app-languages-helm-chart` folder in the Coach Solutions for **Challenge 9** and get it pushed up to your Azure Container Registry
+- First of all we need to allow admin access and authenticate ourselves with the ACR and add it as a helm repository on our local system
```

Comment:
Doing ANYTHING with ACR is the LAST part of this challenge.  Lines 19-69 in this version should stay intact.

You should COPY lines 58 and down and PASTE HERE with the modifications suggested in my next inline comment.